### PR TITLE
changes bash env path to /etc/drydock/.env

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/dry-dock/u14phpall/issues/46

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,3 +3,5 @@ FROM drydock/u14all:master
 ADD . /u14phpall
 
 RUN /u14phpall/install.sh
+
+ENV BASH_ENV "/etc/drydock/.env"
```
Comment (trriplejay) [inline Dockerfile]: @ric03uec @scriptnull why is this needed? i though this line only had to be in the base image (u14)

in this PR: https://github.com/dry-dock/u14javall/pull/56/files  new `.env` items were appended, but the ENV was not added to the Dockerfile, so which implementation is correct?

Comment (scriptnull) [reply Dockerfile]: Oops, I think `BASH_ENV` is already defined in https://github.com/dry-dock/u14/blob/master/Dockerfile#L7 So there is no need for it. I will open a PR to fix this.

7.1 -> 7.2 change is to have the latest version of 7.x PHP. There had already been an effort to make this change in JAN (https://github.com/dry-dock/u14phpall/commit/022320e50d32f3d38b1320c5881dfd38cc53815b ) but we had to revert it ( https://github.com/dry-dock/u14phpall/commit/0793f1e86e614464c6b3ef7fc3de28852afcbd0a ), as one of the existing plugins is not compatible with 7.2.0

This is a similar effort to see if 7.2.2 is compatible with our image. If not, we will revert back to the working 7.1
